database/models/User.ts

```diff
@@ -56,10 +58,16 @@ export class User extends Model {
   @UpdatedAt
   updatedAt!: Date
 
+  @DeletedAt
+  deletedAt?: Date
+
   @BelongsToMany(() => User, {
     onUpdate: "CASCADE",
     onDelete: "CASCADE",
     through: () => SiteMember,
   })
   sites!: Site[]
+
+  @HasOne(() => Site)
+  siteCreated?: Site
```

Comment:
Should this be `HasMany`? A user could have created multiple sites, while a site has only one creator

Reply:
Good catch - let me fix it

Reply:
Fixed in [213be36](https://github.com/isomerpages/isomercms-backend/pull/451/commits/213be369fb65d51f0469fcaf6e841bd6b92328a6)